website_sharing_whole_class_mosaic_plot_translation.Rmd

```diff
@@ -0,0 +1,507 @@
+
+We're trying to create a web site for sharing class resources publicly. 
+In order to give the audiences an overall knowledge of EDA, our website includes the following three parts: 
+
+# Three Parts of Our Website
+
+**1. Course Slides**
+  * We would give a brief introduction to each topics of EDA and provide the slides for further self-learning)
+
+**2. Specific Topic Tutorial**
+  * We are translating the external resources on Mosaic plot topic)
+
+**3. A Reading List**
+  * We would include the recommended reading materials for students to learn certain topics in depth). We have complete the general design of the website and would include more topics during the rest of the semester. 
+
+
+# Links of All Parts of Website
+  * Link to our website: https://leiguolg.wixsite.com/5702 
+    ** The .Rmd and .html files for the 2nd part of our website rpub are also available on the button called ".Rmd and .html files" on the 2nd page of our website named "Specific Topic -- Mosaic plot". These two files can be downloaded through google drive.
+  * Link to translated tutorial: https://rpubs.com/hz2660/Mosaic-plot 
+  * Link to Excel file: resources/website_sharing_whole_class_mosaic_plot_translation/MusicIcecream.csv
+  
+# Codes of 2nd-part of Website 
+  * The .Rmd is available on this Github branch named "Translation-Rcode" 
+
+
+## Note for the Available Code
+ * We used the "Wix" SaaS to generate a website that for sharing class resources publicly, where the code of building the website is not included in the Github since the website is created by "Wix" platform And we include here . This is the second part of our website, which is a specific topic .Rmd tutorial (we are translating the external resources on Mosaic plot topic). 
+
+ * Since we didn't write the code for the Wix website excluding the 2nd part of the website that we wrote with .Rmd, we only include the 2nd part in the Github. 
+ 
+ 
+For graders and Professor: this document is intended as a tutorial in Chinese on the content of Mosaic plots with R. We combined texts from three websites, wiki, *Mosaic plots with ggplot2* (https://cran.r-project.org/web/packages/ggmosaic/vignettes/ggmosaic.html) and the **Chart: Mosaic** section on [edav.info](https://edav.info/). 
+
+We hope this document can effectively jumpstart any user (with limited language background to Chinese) with sufficient skills to assess Mosaic plots with R.
+
+## This document is outlined as follows:
+
+1. We first introduce the defination of the mosaic plot and its usage: what mosaic plots are, what features they have, what aesthetics they have. (This section is adapted from wiki website)
+2. We then describe in detail what the basic parts of plots are. (This section is adapted from edav.info)
+3. Finally, we walked through the conditions representation in mosaic plots, multiple variables in mosaic plots, and orders of variables from *Mosaic plots with ggplot2*.
+
+
+## 1. Introduction 介绍 (Mosaic plot 2019)
```

Comment:
Hashtag headers should not be followed by numbers since the hashtags will create numbered subheadings.